src/components/Board/index.tsx

```diff
@@ -11,25 +25,147 @@ const BoardContainer = styled.div`
   padding: 20px;
 `;
 
-const QuestionTile = styled.div`
-  background-color: #007bff;
+/* const QuestionTile = styled.div` */
+/*   background-color: #007bff; */
+/*   color: white; */
+/*   padding: 20px; */
+/*   text-align: center; */
+/*   cursor: pointer; */
+/*   transition: background-color 0.2s ease-in-out; */
+
+/*   &:hover { */
+/*     background-color: #0056b3; */
+/*   } */
+/* `; */
+
+// 
+const CategoriesContainer = styled.div`
+  display: grid;
+  grid-template-columns: repeat(5, 1fr);
+  gap: 10px;
+  padding: 20px;
+`;
+
+const CategoryTile = styled.div`
+  background-color: #0056b3;
   color: white;
   padding: 20px;
   text-align: center;
-  cursor: pointer;
-  transition: background-color 0.2s ease-in-out;
-
-  &:hover {
-    background-color: #0056b3;
-  }
 `;
 
 export const Board = () => {
+  const [isAnswerShown, setIsAnswerShown] = useState<boolean>(false);
+  const [selectedQuestion, setSelectedQuestion] = useState<Question | null>(
+    null
+  );
+  const [team1Score, setTeam1Score] = useState<number>(
+    Number(localStorage.getItem("team1Score")) || 0
+  );
+  const [team2Score, setTeam2Score] = useState<number>(
+    Number(localStorage.getItem("team2Score")) || 0
+  );
```

Comment:
You can use nullish coalescing for both states like this: 

`Number(localStorage.getItem("team1Score")) ?? 0`

`Number(localStorage.getItem("team2Score")) ?? 0`